internal/fs/fs.go

```diff
@@ -270,9 +271,19 @@ func CopyDir(src, dst string) error {
 // the copied data is synced/flushed to stable storage.
 func copyFile(src, dst string) (err error) {
 	if sym, err := IsSymlink(src); err != nil {
-		return err
+		return errors.Wrap(err, "symlink check failed")
 	} else if sym {
-		return cloneSymlink(src, dst)
+		if err := cloneSymlink(src, dst); err != nil {
+			if runtime.GOOS == "windows" && strings.HasSuffix(err.Error(), "A required privilege is not held by the client.") {
```

Comment:
I solved think with depending on an OS-independent way. Let me know if a windows-tagged file is still needed.